src/tcp_ip/stream_follower.cpp

```diff
@@ -84,7 +84,7 @@ void StreamFollower::process_packet(PDU& packet, const timestamp_type& ts) {
     if (iter == streams_.end()) {
         // Start tracking if they're either SYNs or they contain data (attach
         // to an already running flow).
-        if (tcp->flags() == TCP::SYN || (attach_to_flows_ && tcp->find_pdu<RawPDU>() != 0)) {
+        if (tcp->has_flags(TCP::SYN) || (attach_to_flows_ && tcp->find_pdu<RawPDU>() != 0)) {
```

Comment:
Minor but I'd probably create a variable like

```c++
const bool is_syn = tcp->has_flags(TCP::SYN);
```

And then use it in both conditions below.

Reply:
It is up to you, if you think that code will be more readable. Or you are worry about performance?

Reply:
Not really performance related, just so we avoid having the same expression twice.